content/docs/matrix/_index.md

```diff
@@ -83,13 +83,13 @@ The table shows the protocol each system uses, examples of resources you'd acces
       <th>The provider uses…</th>
       <td>Apache, Nginx</td>
       <td>sendmail, postfix</td>
-      <td>Ruma, Synapse</td>
+      <td>Conduit, Synapse, Dendrite</td>
     </tr>
     <tr>
       <th>You use…</th>
       <td>Chrome, Firefox</td>
       <td>Gmail web app, Apple Mail</td>
-      <td>Riot, WeeChat</td>
+      <td>Element, Fractal, Daydream</td>
```

Comment:
Not sure whether to mention Daydream (cc @MTRNord). In any case, WeeChat can still be used, there's even a rewrite of the Matrix integration [in Rust](https://github.com/poljar/weechat-matrix-rs).

```suggestion
      <td>Element, Fractal, WeeChat</td>
```

Reply:
Yeah probably daydream isnt something that should be mentioned at this point as development is paused (waiting on dependencies)

Reply:
😢

Reply:
I plan to start trying to get druid version updated about next weekend ;) So there will be development in the future